app/src/main/java/com/example/androidacademy/db/entities/MoviesDatabase.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.androidacademy.db.entities
+
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.example.androidacademy.App
+
+@Database(entities = [MovieEntities::class, ActorsEntites::class], version = 1, exportSchema = false)
+abstract class MoviesDatabase : RoomDatabase() {
```

Comment:
Можно заменить на интерфейс. Тогда можно будет избавиться от abstract при объявлении методов.
```suggestion
interface MoviesDatabase : RoomDatabase() {
```